### ISSUE TITLE
Wandering Isle Province

### ISSUE THREAD

**Issue by ercarp** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#686]
**Describe your suggestion in full detail below:**
I think we should add the [Wandering Isle](https://wowpedia.fandom.com/wiki/Wandering_Isle) (Shen-zin Su) as a province on the map. It won't be _wandering_, but it would still be better than not having it at all.

And besides, there are ways to simulate the island's absence via gameplay mechanics:
- A. Make the island only interactable at specific intervals. Maybe once every two years or something.
  - Basically the same thing we are planning to do with Pandaria and its mists, except in this case the island would disappear again after a while and then return later at a different time.
- B. Or we could have the island "appear" (become accessible) at random times, quite rarely, which would trigger an event for the player.

For the location, I think almost anywhere is fine, but perhaps the best option would be somewhere near Pandaria.

**Comment by Grekopithikos** [comment]
I'm not sure if there is a need to make the Wandering Isle a province in the map.
You could just try making it a "landless" title that "travels" through certain points of the map, and when it supposedly comes close to the place an event fires making it interactible to the characters there.

**Comment by zumbak04** [comment]
I wouldn't add it as a province too. Instead, add events for Pandaria when Shen-zin Su returns back.

**Comment by ercarp** [comment]
> I wouldn't add it as a province too. Instead, add events for Pandaria when Shen-zin Su returns back.
> You could just try making it a "landless" title that "travels" through certain points of the map, and when it supposedly comes close to the place an event fires making it interactible to the characters there.

@Grekopithikos @zumbak04 You both make excellent points and I think I agree. Should I just close this issue then? Or maybe edit it to be about adding Shen-zin Su events instead?

**Comment by zumbak04** [comment]
> @Grekopithikos @zumbak04 You both make excellent points and I think I agree. Should I just close this issue then? Or maybe edit it to be about adding Shen-zin Su events instead?

@ercarp I would edit.

**Comment by ercarp** [comment]
> 
> 
> > @Grekopithikos @zumbak04 You both make excellent points and I think I agree. Should I just close this issue then? Or maybe edit it to be about adding Shen-zin Su events instead?
> 
> @ercarp I would edit.

Done.
